SPEC.md

```diff
@@ -190,8 +190,8 @@ In this case, the executor will bind mount the host's `/opt/tenant1/work` direct
 
 #### Network Setup
 
-A Pod must have a [layer 3](http://en.wikipedia.org/wiki/Network_layer) (commonly called the IP layer) network interface, which can be instantiated in any number of ways (e.g. veth, macvlan, ipvlan, device pass-through).
-The network interface MUST be configured with an IPv4/IPv6 address that is reachable from other pods.
+A Pod must have a loopback network interface and one or more [layer 3](http://en.wikipedia.org/wiki/Network_layer) (commonly called the IP layer) network interfaces, which can be instantiated in any number of ways (e.g. veth, macvlan, ipvlan, device pass-through).
```

Comment:
I would actually require zero or more non-loopback interfaces. It provides a possibility to have a pod be "off the network". This is problematic today as it would make it impossible to reach metadata service but since we want to make it optional, it should not be an issue.